game/src/edit/traffic_signals/mod.rs

```diff
@@ -651,14 +672,20 @@ fn make_side_panel(
                         ),
                     }
                     .draw_text(ctx),
-                    Btn::svg_def("system/assets/tools/edit.svg").build(
-                        ctx,
-                        format!("change duration of stage {}", idx + 1),
-                        if selected == idx { Key::X.into() } else { None },
-                    ),
+                    {
+                        let mut button = ctx
+                            .style()
+                            .btn_plain_light_icon("system/assets/tools/edit.svg");
```

Comment:
Er, this SVG is showing up as the info icon for me?
![Screenshot from 2021-01-21 11-14-09](https://user-images.githubusercontent.com/1664407/105400450-cb31c700-5bd9-11eb-8bf8-2fee28ed3d99.png)

Reply:
I remember the copy and paste haze induced moment that this happened - thank you!

I also now realize that there is a new delete asset - so I'm using that, along with the standard button styling (and tweaked the alignment/spacing):

<img width="492" alt="Screen Shot 2021-01-21 at 2 15 22 PM" src="https://user-images.githubusercontent.com/217057/105407271-1c01e980-5bf3-11eb-8d10-1b65e782e08d.png">